libs/utils/README.md

```diff
@@ -14,9 +14,9 @@ npm install @studiohyperdrive/ngx-utils
 
 This package will follow a semver-like format, `major.minor.patch`, in which:
 
-- `major`: Follows the Angular major version
-- `minor`: Introduces new features and (potential) breaking changes
-- `patch`: Introduces bugfixes and minor non-breaking changes
+-   `major`: Follows the Angular major version
```

Comment:
Is this really necessary? Because now we get an inconsistency between the readmes 🙈 If it's needed it's better to just do them all at once in a separate PR.

Reply:
It is not, this was auto changed. I'll revert